distribution/changelog.txt

```diff
@@ -42,6 +42,7 @@
 - Fix: [#22121] Some news items in the “Recent Messages” window have the wrong text colour.
 - Fix: [#22152] [Plugin] Negative signed integers are truncated.
 - Fix: [#22161] Using arrow keys in textboxes crashes the game.
+- Fix: [#22173] Patrol path selection not visible over existing patrol paths.
```

Comment:
Probably more of a `Change` than a `Fix`, right?

Reply:
I think so too.

Reply:
I've updated it